test/route_lister.py

```diff
@@ -26,17 +26,13 @@ def get_doc(handler, method):
 
   routes=[]
   for i in get_routes(router):
-    if hasattr(i, 'handler'):
-      # I think this means it's a route, not a path prefix
-      cur_template = i.template
-      cur_handler  = i.handler
-      cur_method   = i.handler_method
-      cur_doc      = get_doc(cur_handler,cur_method)
-      r={'template':cur_template, 'handler':cur_handler, 'method':cur_method, 'doc':cur_doc, 'name': i.name}
-      routes.append(r)
-    else:
```

Comment:
I replaced the else here with an assert since I didn't see the block of code entered

Reply:
Cool I can leave it in then, will be easier to cross the bridge when and if we come to it.